Data-Science/Feast/definitions.py

```diff
@@ -25,7 +25,7 @@
 # for more info.
 driver_stats_source = FileSource(
     name="driver_hourly_stats_source",
-    path="/home/hpedemouser01/feast/data/driver_stats.parquet",
+    path="/home/hpedemo-user01/user/Feast/data/driver_stats.parquet",
```

Comment:
We should probably parameterize the hpedemo-user01, it can be different user. The path depends on the user name.

Reply:
@CybervisionVolodymyrPogorelov done. thx